src/modules/LanItems.rb

```diff
@@ -2177,6 +2172,8 @@ def Commit
     # Remove a half-configured item.
     # @return [true] so that this can be used for the :abort callback
     def Rollback
+      return true if LanItems.operation != :add
```

Comment:
I was testing it, and if you edit a not configured interface and then :abort it currently crash

Reply:
hm, that is close to use case which i was fixing. I'll try even this one. thx

Reply:
You can split it in different pull requests or, in case you decide to fix also this issue as part of this pull request, then cover also the scenario with unit tests.

Reply:
seems that subsequent commit fixed even "my case" so dropping this change is enough. It is nice how is networking code straightforward and robust ;-) thanks for co-testing